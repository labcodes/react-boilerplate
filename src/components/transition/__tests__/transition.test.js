'use strict';

import React from 'react';

// Test helpers
import renderer from 'react-test-renderer';

// Component
import Transition from 'components/transition';

// ====

describe('Transition [Snapshot]', () => {
    it('Renders correctly', () => {
        const tree = renderer.create(<Transition />).toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('Renders correctly', () => {
        const tree = renderer.create(<Transition customClass="ds" />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});

