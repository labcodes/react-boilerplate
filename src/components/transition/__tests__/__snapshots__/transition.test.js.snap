// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Transition [Snapshot] Renders correctly 1`] = `
<section
  className="page-wrapper "
  style={
    Object {
      "opacity": 0,
      "transition": "opacity 150ms ease-in-out",
    }
  }
/>
`;
